openeo/rest/datacube.py

```diff
@@ -308,7 +308,7 @@ def filter_bbox(
                 >>> cube.filter_bbox(bbox_param)
                 >>> cube.filter_bbox(bbox=bbox_param)
 
-            - With a CRS other than EPSG 4326::
+            - With a CRS other than EPSG 4326 (**see also parameters: crs**)::
```

Comment:
putting something in bold (more important) between parentheses (less important) looks weird :smile: 

I don't think this clarification is necessary at all.
Maybe it's enough to reformat the snippet a bit to highlight the crs parameter, e.g.

```
>>> cube.filter_bbox(
...     west=652000, east=672000, north=5161000, south=5181000, 
...     crs=32632
... )
```

Reply:
Done, reformatted as suggested.